src/CreeDictionary/CreeDictionary/templates/CreeDictionary/components/citations.html

```diff
@@ -0,0 +1,46 @@
+{% spaceless %}
+
+  {% comment %}
+    Citations for a definition.
+
+    Parameters:
+      - dictionary_sources: list[str] — a list of dictionary source IDs (abbreviations)
+
+    Outputs a list of citations with bibliographic information in the tooltip.
+    Currently this bibliographic citation is **hardcoded into this template**
+    for crkeng (itwêwina: the Plains Cree Dictionary). The references are in
+    MLA format; however, I decided to hardcode them because there are
+    exactly TWO sources and they both have subtly different MLA citation
+    format 😖
+
+    Example:
+
+       [CW] [MD]
+        ^
+      [ Wolvengrey, Arok, editor. /Cree: Words/. 2001. ]
+  {% endcomment %}
+
+  {% load creedictionary_extras %}
+
+  {% for source in dictionary_sources %}
+    {% with id=request|unique_id %}
+      <cite class="cite-dict" data-has-tooltip aria-describedby="tooltip:{{ id }}">{{ source }}</cite>
+
+      <div id="tooltip:{{ id }}" class="tooltip" role="tooltip" data-cy="citation-tooltip">
+        {% if source == "CW" %}
+          Wolvengrey, Arok, editor.
+          <span class="source-title">Cree: Words</span>.
+          Regina, University of Regina Press, 2001.
+        {% elif source == "MD" %}
+          <span class="source-title">Maskwacîs Dictionary</span>.
+          Maskwacîs, Maskwachees Cultural College, 1998.
```

Comment:
These bibliography entries are in MLA format. Annoyingly, they both have a different format. I think the original intent with the `DictionarySource` model and all of its fields was that there would be some sort of bibliography entry formatter that would intelligently format any given source in the citation format required. But this is weird complexity for exactly two dictionary sources. I'm not even sure the other language pairs have this problem, so for now, all of the formatting and required information is simply hard-coded here.